Add OPERA RTC as job type (#342)
ASFHyP3/hyp3-sdk: CHANGELOG.md

and this project adheres to [PEP 440](https://www.python.org/dev/peps/pep-0440/)
and uses [Semantic Versioning](https://semver.org/spec/v2.0.0.html).

## [7.4.0]

### Added

> * Added `HyP3.submit_opera_rtc_s1_job` and `HyP3.prepare_opera_rtc_s1_job` methods for submitting OPERA RTC-S1 jobs to HyP3.

Reviewer: Let's add a note that the job type is only available in https://hyp3-test-api.asf.alaska.edu

> * Added `HyP3.ServiceUnavailableError` for handling 503 errors from HyP3 .

Reviewer: ```suggestion
* Added `hyp3_sdk.exceptions.ServiceUnavailableError` for handling 503 errors from HyP3 .
```

> * Added `HyP3.submit_opera_rtc_s1_job` and `HyP3.prepare_opera_rtc_s1_job` methods for submitting OPERA RTC-S1 jobs to HyP3. This job type is under development and is currently only be availble in [HyP3 test](https://hyp3-test-api.asf.alaska.edu)

Reviewer: Providing the test url that would be passed into the `HyP3` constructor is probably the most useful:
```suggestion
* Added `HyP3.submit_opera_rtc_s1_job` and `HyP3.prepare_opera_rtc_s1_job` methods for submitting OPERA RTC-S1 jobs to HyP3. This job type is under development and is currently only available in `https://hyp3-test-api.asf.alaska.edu`
```

> * Added `hyp3_sdk.exceptions.ServiceUnavailableError` for handling 503 errors from HyP3 .

Reviewer: ```suggestion
* Added `hyp3_sdk.exceptions.ServiceUnavailableError` for handling 503 errors from HyP3.
```
